src/server/models/Player.js

```diff
@@ -463,11 +470,9 @@ export default class Player extends Model {
   }
 
   async cleanup() {
-    let card;
-    while (card = this.playArea.find(c => !c.ignoreCleanUp)) {
-      await this.discard(card, this.playArea);
-    }
-    await this.discardAll(this.hand);
+    await this.discardAll([...this.hand]);
```

Comment:
nvm, you are one step ahead of me. Not sure if we'll ever care which is discarded first, hand or play area.

Reply:
Yeah this was specifically for travellers - giving the option of choosing which one to exchange first